UI/styles.css

```diff
@@ -0,0 +1,354 @@
+body{
+	margin: 0;
+}
+header {
+	background-color: black;
+	display: flex;
+	justify-content: space-between;
+	height: 90px;
+	align-items: center;
+	position: fixed; 
+	top: 0;
+	width: 100%;
+}
+	.header__logo {
+		color: white;
+		padding: 0 2%;
+		font-size: 60px;
+		font-family: "Myriad Pro",sans-serif;
+
+	}
+	.header__user-menu {
+		display: flex;
+		/*align-items: flex-end;*/
+		justify-content: space-around;
+		padding: 0 5%;
+		font-size: 24px;
+		width: 500px;
+		align-items: center;
+	}
+		.header__button-add {
+			/*padding: 0 1%;*/
+
+		}
+			.button-add {
+				background-color: #00C5C0;
+				width: 160px;
+				height: 50px;
+				border-radius: 30px;
+				border: 0;
+				font-size: 24px;
+			}
+
+		.header__name {
+			color: white;
+			font-size: 31px;
+			/*padding: 0 1%;*/
+		}
+		.header__log-out {
+			color: #9F2214;
+			font-size: 24px;
+			/*padding: 0 1%;*/
+		}
+
+		
+footer {
+	background-color: black;
+	color: white;
+	text-align: center;
+	/*position: fixed; */
+	/*left: 0; bottom: 0;*/
+	width: 100%;
+
+}
+	.footer__text{
+		margin: 0;
+		padding: 0.5%;
+		font-size: 70%;
+	}
+main {
+	margin: 90px 0 0 0;
+	display: flex;
+	height: 100%;
+}
+.filter-table {
+	/*position: fixed;*/
+	width: 30%;
+	/*height: 100%;*/
+	background-color: #A8A8A8;
+	
+}
+	.filter-tools {
+		width: 83%;
+		margin: 15% 6% auto auto; 
+		font-size: 24px;
+		font-family: "Verdana";
+		
+	}
+		.filter-tools__date {
+			padding: 10px 0;
+			display: flex;
+			justify-content: space-between;
+			align-items: center;
+		}
+			.input-date{
```

Comment:
Your code editor most likely has some code formatting features. Please google them and use when you write code.

Reply:
If you're talking about tabs. I used this to make it easier to navigate the structure of the classes. 
Can I do that?